packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -236,6 +236,7 @@ export class Summarizer extends EventEmitter implements ISummarizer {
             eventName: "RunningSummarizer",
             onBehalfOf,
             initSummarySeqNumber: this.runtime.deltaManager.initialSequenceNumber,
+            ...this.configurationGetter(),
```

Comment:
Not sure if this is the best approach, but I'm not able to just put any object into a telemetry event. Each property has to be either `TelemetryEventPropertyType` or `ITaggedTelemetryPropertyType`, which are mostly limited to primitive types

Reply:
I think @markfields started to put details for that, like here:

    return PerformanceEvent.timedExecAsync(
        logger, {
            eventName: "JoinSession",
            attempts: options.refresh ? 2 : 1,
            details: JSON.stringify(details),
            ...tokenRefreshProps,
        },